app/javascript/components/wallet/TransactionList.js

```diff
@@ -12,8 +12,8 @@ export default class TransactionList extends Component {
   constructor(props) {
     super(props);
     let botCoin = new BotCoin()
-    let registry = new DeveloperRegistry(window.app_config.developer_registry_contract);
-    let bot_registry = new BotRegistry(window.app_config.bot_registry_contract);
+    let registry = new DeveloperRegistry(window.keyTools.currentNetworkConfig.developer_registry_contract);
+    let bot_registry = new BotRegistry(window.keyTools.currentNetworkConfig.bot_registry_contract);
```

Comment:
This is about all classes from 'blockchain' folder. You get the address in each constructor and here while constructor takes 0 arguments. Apparently, everything works, but we should be able to put empty braces here and it will work as well.